src/cluster/Cargo.toml

```diff
@@ -42,7 +42,7 @@ proclist = "0.9.2"
 remoteprocess = "0.4.2"
 
 # Fluvio dependencies
-fluvio = { version = "0.6.0", path = "../client", default-features = false }
+fluvio = { version = "0.6.1-alpha.1", path = "../client", default-features = false }
```

Comment:
Is there reason need newer client api?

Reply:
Since we also specify the `path` dependency as well, it requires you to use an exact matching version as what is in the repo.

Reply:
Yuck.  I guess we can revert back once we published regular version